app/org/sagebionetworks/bridge/models/DateRange.java

```diff
@@ -0,0 +1,57 @@
+package org.sagebionetworks.bridge.models;
+
+import javax.annotation.Nonnull;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.datatype.joda.deser.LocalDateDeserializer;
+import org.joda.time.LocalDate;
+
+import org.sagebionetworks.bridge.json.LocalDateToStringSerializer;
+
+/** Model object representing a date range, which includes start date and end date as calendar dates (YYYY-MM-DD). */
+public class DateRange implements BridgeEntity {
+    private final @Nonnull LocalDate startDate;
+    private final @Nonnull LocalDate endDate;
+
+    /**
+     * DateRange constructor. Also validates that startDate and endDate are specified and that startDate isn't after
+     * endDate.
+     *
+     * @param startDate
+     *         start date, must be non-null
+     * @param endDate
+     *         end date, must be non-null
+     */
+    public DateRange(
+            @Nonnull @JsonProperty("startDate") @JsonDeserialize(using = LocalDateDeserializer.class) LocalDate startDate,
```

Comment:
Are the deserializers necessary, they shouldn't be if you use BridgeObjectMapper has the JodaModule registered with it. Do we use it to deserialize JSON in the controllers?